### ISSUE TITLE
Unable to reproduce the published mAP for example/ssd with VGGNET model VOC0712  data

### ISSUE THREAD

**Issue by ajayvohra2005** [apache/incubator-mxnet#9622]
I was not able to reproduce the published mAP with SSD VGG16_reduced model  with VOC0712 train-validation and VOC07 test data (300x300 image size) with MxNet (Expected 0.77, Actual 0.73).

I used an effective batch size of 32 on 2 GPUs with a learning rate of 0.001. Rest of the hyper-parameters were as documented in GitHub. 

Trained on AWS Deep Learning AMI Ubuntu Linux - 2.4_Oct2017 (ami-37bb714d) p2.8xlarge

[ssd_caffe_test.txt](https://github.com/apache/incubator-mxnet/files/1675704/ssd_caffe_test.txt)
[ssd_caffe_train.txt](https://github.com/apache/incubator-mxnet/files/1675705/ssd_caffe_train.txt)
[ssd_mxnet_trainval.txt](https://github.com/apache/incubator-mxnet/files/1675706/ssd_mxnet_trainval.txt)

**Comment by sunyonggang** [comment]
I trained the example with all default params, but gpu only  2. 
the example `VGG16_reduced 300x300` shows 0.778, but I get only 0.72 with epoch 100.
then, I change learning_rate to 0.001, mAp only 0.71 with epoch 200.-_-

**Comment by ThomasDelteil** [comment]
I confirm this is still the case:
`INFO:root:Epoch[239] Validation-mAP=0.709660`

**Comment by stu1130** [comment]
The model is out-of-date. @ajayvohra2005 @sunyonggang you might want to use the [gluon-cv](https://gluon-cv.mxnet.io/build/examples_detection/train_ssd_voc.html) to train the latest SSD model
